ethcore/src/blockchain/blockchain.rs

```diff
@@ -587,21 +586,21 @@ impl BlockChain {
 
 	/// This functions returns modified blocks blooms.
 	///
-	/// To accelerate blooms lookups, blomms are stored in multiple 
-	/// layers (BLOOM_LEVELS, currently 3). 
+	/// To accelerate blooms lookups, blooms are stored in multiple
```

Comment:
though this time small, in general, this change is irrelevant to this pr and should go in the separate one